projects/components/src/popover/popover.service.ts

```diff
@@ -78,4 +78,20 @@ export class PopoverService {
       ])
     );
   }
+
+  private hasBackdrop(backdrop?: PopoverBackdrop): boolean {
+    return backdrop === PopoverBackdrop.Opaque || backdrop === PopoverBackdrop.Transparent;
+  }
+
+  private getBackdropClass(backdrop?: PopoverBackdrop): string {
+    switch (backdrop) {
+      case PopoverBackdrop.Transparent:
+        return 'cdk-overlay-transparent-backdrop';
```

Comment:
not sure what's in this, but does it make sense to migrate to our own class for consistency? how much can a transparent backdrop have in it?

Reply:
I don't think there is any benefit. It sets the opacity. I think we can leave it as it is till we modify the style.